lib/bundler.js

```diff
@@ -7,6 +7,7 @@ const browserify = require('browserify');
 const babelify = require('babelify');
 
 const bes2015 = require('babel-preset-es2015');
+const protoAssign = require('babel-plugin-transform-object-set-prototype-of-to-assign');
```

Comment:
I think there is a babel polyfill that apex is using that might fix all of the issues